src/main/java/core/basesyntax/service/impl/ReportWriterCsv.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriterCsv implements ReportWriter {
```

Comment:
```suggestion
public class WriterImpl implements Writer { //or CommonWriter or think about better name
```

Reply:
Not resolved, your writer is not related to csv extension, you can write any file you want